card/mod/machines/set/all/reset_machines.rb

```diff
@@ -2,8 +2,11 @@ module ClassMethods
   def reset_script_machine
     Auth.as_bot do
       card = Card[:all, :script, :machine_output]
-      card.update_columns trash: true
-      card.expire
+      if card
```

Comment:
if there's not a card, reset won't create it?

Reply:
a cyp failed because there was no card